packages/flutter/lib/src/material/data_table_theme.dart

```diff
@@ -46,7 +54,9 @@ class DataTableThemeData with Diagnosticable {
     this.columnSpacing,
     this.dividerThickness,
     this.checkboxHorizontalMargin,
-  });
+  }) :
+    _dataRowHeight = dataRowHeight,
+    _dataRowHeightSettings = dataRowHeightSettings;
```

Comment:
you can initialize _dataRowHeightSettings from dataRowHeight directly so you don't need _dataRowHeight

Reply:
Unfortunately, if I want to keep the `DataTableThemeData` constructor `const`, then I cannot create `DataRowHeightStyle` in the initializer list like `const DataRowHeightStyle.fixed(height: dataRowHeight)` because "const Constructor parameters (here dataRowHeight) are never considered const" (see dart-lang [#2000](https://github.com/dart-lang/language/issues/2000))

So I do not see another solution.